apps/full-site-editing/full-site-editing-plugin/full-site-editing/editor/style.scss

```
@@ -2,4 +2,22 @@
 	.editor-post-title {
 		display: none;
 	}
+
+	.edit-post-visual-editor {
+		margin-top: 20px;
+		padding-top: 0;
+	}
+}
+
+.post-type-page, .post-type-wp_template {
+	.edit-post-layout__content {
+		background: #eee;
+	}
+	
+	.edit-post-layout__content .edit-post-visual-editor {
+		flex: none;
+		margin: 60px;
+		box-shadow: 0 0 30px 0 rgba(0,0,0,0.1);
+		background: #fff;
+	}
 }
```

```suggestion
}

```

Missing newline at the end of file.